apps/web/pages/cancel/[uid].tsx

```diff
@@ -81,9 +81,13 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
                         </h3>
                         <div className="mt-2">
                           <p className="text-center text-sm text-gray-500">
-                            {props.cancellationAllowed && !props.booking?.eventType.recurringEvent
-                              ? t("reschedule_instead")
-                              : t("event_is_in_the_past")}
+                            {!props.booking?.eventType.recurringEvent
+                              ? props.cancellationAllowed
+                                ? t("reschedule_instead")
+                                : t("event_is_in_the_past")
+                              : allRemainingBookings === "true"
+                              ? t("cancelling_all_recurring")
+                              : t("cancelling_event_recurring")}
```

Comment:
We are now relying on knowing if the cancellation is just for an event in a series or the entire series to show correct messaging when the user wants to cancel a booking.